### ISSUE TITLE
GPU requirements

### ISSUE THREAD

**Issue by niccanicci** [andabi/deep-voice-conversion#22]
hello @andabi , does this model need to run on a higher GPU. My gpu is nvidia geforce gtx1080 with 8G. is it enoug to run? if its doesnt can i change the batchsize and decrease the dataset. Thank you

**Comment by JiaYK** [comment]
By default, you can run the demo. 
My gpu is tesla k80 with 12G.
And in train2, I can use most 600m wav files.

**Comment by VictoriaBentell** [comment]
I'm using that same gpu and it's working fine for me without changing the batch size or changing anything with the dataset.

**Comment by niccanicci** [comment]
why mine stuck when load the train1.py? i waited more than one hour and still dont get any result? @VictoriaBentell

**Comment by VictoriaBentell** [comment]
I'm assuming you're using the TIMIT dataset. Check line 49 in hparams.py and make sure that you have the correct path. Also make sure that the files in your dataset are all .wav instead of .WAV. I also had to move everything out of the 'raw' folder to have the correct path.

**Comment by VictoriaBentell** [comment]
Put this into the deep-voice-conversion folder, change it to rename_WAV.py (because I can't upload python files here) and run it to change all of the .WAV files to .wav
Then change `os.chdir('./datasets/timit/TIMIT/TRAIN')` to `os.chdir('./datasets/timit/TIMIT/TEST')` in line 2 and run it again, so it does this for both the train and test folders.

[rename_WAV.txt](https://github.com/andabi/deep-voice-conversion/files/1801057/rename_WAV.txt)

**Comment by hoenickf** [comment]
Hi, 
thanks for being so responsive.
Even after renaming the files, I still have the same problem.
Also i triple-checked if the destination folder is correct.
Do you have any idea?
![image](https://user-images.githubusercontent.com/11627845/37418674-0e8ebf98-27b3-11e8-834c-aaf4031ec0a9.png)

**Comment by VictoriaBentell** [comment]
@hoenickf do you think you can post your hparams.py?

**Comment by hoenickf** [comment]
I did not make any changes at all. So it would be kind of boring to look at :)

**Comment by VictoriaBentell** [comment]
This is a great repo and all, but it would be nice if it gave an error when something is wrong. Maybe the script I wrote sucks and didn't change all of the file names? Otherwise if you pulled everything out of the 'raw' folder, I'm not sure what could be the case. :/

**Comment by vivek-sethia** [comment]
@VictoriaBentell I am using **Tesla K80** and my **train2.py** with dataset pointed to **arctic/slt** dataset.
script is processing but very slowly. Any ideas?

**Comment by vivek-sethia** [comment]
@VictoriaBentell I am using  **Tesla K80**. **train2.py** points to **arctic/slt dataset** and I have reduced the number of epochs to **500** also, but I have kept other parameters intact. But it is taking a lot of time to process. Can you tell me if there is something that needs to be checked?

**Comment by VictoriaBentell** [comment]
I'm having problems with train2.py as well, since it doesn't seem to be making any progress at all my setup. It just fluctuates between high and low loss all day long without any clear pattern. :/

**Comment by vivek-sethia** [comment]
@VictoriaBentell Thats same here for me. Hope someone who has executed the whole process can share his views over here.

**Comment by guang** [comment]
have yall checked your `nvidia-smi` to make sure the GPU is actually running? in my experience it will default to `tensorflow` instead of `tensorflow-gpu` unless i uninstall `tensorflow` and reinstall `tensorflow-gpu` and then it seems to work
ill try to PR a dockerfile soon
